src/mon/Monitor.cc

```diff
@@ -536,14 +536,14 @@ int Monitor::preinit()
   assert(!logger);
   {
     PerfCountersBuilder pcb(g_ceph_context, "mon", l_mon_first, l_mon_last);
-    pcb.add_u64(l_mon_num_sessions, "num_sessions");
-    pcb.add_u64_counter(l_mon_session_add, "session_add");
-    pcb.add_u64_counter(l_mon_session_rm, "session_rm");
-    pcb.add_u64_counter(l_mon_session_trim, "session_trim");
-    pcb.add_u64_counter(l_mon_num_elections, "num_elections");
-    pcb.add_u64_counter(l_mon_election_call, "election_call");
-    pcb.add_u64_counter(l_mon_election_win, "election_win");
-    pcb.add_u64_counter(l_mon_election_lose, "election_lose");
+    pcb.add_u64(l_mon_num_sessions, "num_sessions", "Current number of opened monitor sessions");
+    pcb.add_u64_counter(l_mon_session_add, "session_add", "Number of created monitor sessions");
+    pcb.add_u64_counter(l_mon_session_rm, "session_rm", "Number of remove_session calls in monitor");
+    pcb.add_u64_counter(l_mon_session_trim, "session_trim", "Number of triming monitor sessions (eg because of timeout)");
```

Comment:
s/triming/trimmed/ and I would remove the **(eg...)** because someone reading this number and trying to make sense of it would need a lot more info to know what it means